src/IO/WriteBufferFromS3TaskTracker.h

```diff
@@ -0,0 +1,37 @@
+#pragma once
+
+#include "config.h"
+
+#if USE_AWS_S3
+
+#include "WriteBufferFromS3.h"
+
+namespace DB
+{
+
+class WriteBufferFromS3::TaskTracker
```

Comment:
Could we move this class to `WriteBufferFromS3.cpp`? Because this class doesn't seem to be needed in a header.